### ISSUE TITLE
Get Environment add additional data

### ISSUE THREAD

**Issue by dustinvanbuskirk** [codefresh-io/cli#170]
Right now we only return environment name and id.

I need to see the additional data below when querying by id.

Environment State (Running/Paused/Terminating)
Environment URL (Full URL including special port)
Time Running
Services
  - Service Name
     - Docker image/tag
     - GIT SHA
     - GIT Branch
     - Service URL (Any service specific URLs if we decide to expose more than one service)

Basically anything we disclose currently from the Environments page in the UI when we look for additional information on an environment
